blazor/datagrid/data-binding.md

```diff
@@ -1198,6 +1198,90 @@ The following sample code demonstrates notifying user when server-side exception
     }
 }
 ```
+
+## Performing DataGrid operations in server side
+
+You can get the data from a database and bind it to the DataGrid using the WepAPI service. This can be achieved by the WebApiAdaptor adaptor of SfDataManager. The DataGrid operations (sorting, filtering, paging, and searching) performed on the server side using "Request.Query" from the Get method of the controller class. This "Request.Query" contains all the required parameters for performing server-side operations such as paging, sorting, and filtering. 
```

Comment:
While using the WebAPI adaptor type in the SfDataManager component, we will generate queries for data operations to send to the server side as query string and based on these query string values, actions will be performed in server side. 

Queries sent to the server can be retrieved using QueryString and these values can be differentiated as below. create table or points using $skip, $top, $filter and $search , $sort along with the explanation. 

using the above querystring keys, we can get the corresponding values and perform the required actions. in the below example, we have demonstrated the simple filtering, paging and sorting operation. 
Similarly we suggest you to handle the scenarios for complex queries.